### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by Nooria150**
1. At first cloned my repo.
2. I followed the Gitlow.
3. created feature branch.
4. setup linters.
5. added readme.md file.
6. started coding for my portfolio website.
7. then I designed it.

### CODE REVIEW

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Nooria150 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although you have passed the project requirement, but your code is not clean and also your portfolio links are all directed to the github, give the exact links for all of them.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Personal Portfolio and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌Your code is clean
✔️Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Nooria150** [comment]
thank you so mush sir for the review. I Implemented the feedback. I check it from every view It is correct, my partner is not  capable to check my website. so that's why I wanna tag you again. @ahzamir

**Comment by ahzamir** [comment]
@Nooria150 

- You have to tag her for review and feedback if she is capable or not doesn't matter and also remind her in the group to provide you review and feedback.

- If she did not provide you feedback within 24 hours then you can ask me to provide you feedback.
  - But don't forget that you have to request me after you implemented the best version of your code it means you were sure and checked everything.

**Comment by Nooria150** [comment]
@ahzamir 
sure sir! can I tag her they same way I tagged you? I mean her email address name?

**Comment by Nooria150** [comment]
@ahzamir 
sorry! I think I should tag her github name.

**Comment by Breshnahamidi20** [comment]
Hi @Nooria150
I want to review your code and clone it to my local machine, but there is something wrong with the link.

**Comment by ahzamir** [comment]
@Breshnahamidi20 

- You have to use her ssh project link inorder to clone her project and check.

**Comment by Nooria150** [comment]
@ahzamir please check my project. coz tomorrow is deadline. And tomorrow is my brother's wedding. I have tried my best to  make it the best project but when I saw some of girls project I think mine is not enough good. sorry for disappointed you. 
I think I still have to learning more about CSS. this would not be enough for making good website.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  </div>
+  <div class="contact__links">
+    <a href="https://github.com/Nooria150" id="profile-link" target="_blank" rel="noopener">Github</a>
+    <a href="nooriamangal55@gmail.com" id="profile-link" target="_blank">Send an Email</a>
+    <a href="555 555 555" id="profile-link" target="_blank">Call me</a>
+  </div>
+</section>
+<footer class="footer">
+  <p class="wrap inline">This portfolio is for practic.</p>
+</footer>
```
Comment (ahzamir) [inline index.html]: - These codes all are inside your body si they all should be 2 spaces ahead of your body tag

**Review by ahzamir — CHANGES_REQUESTED** [review]

**Comment by Nooria150** [comment]
@ahzamir I implemented the feedback.

**Review by ahzamir — APPROVED** [review]
### Hi @Nooria150 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Personal Portfolio project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your project has a professional README.md file.

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**